modules/SofaGeneralAnimationLoop/MechanicalMatrixMapper.h

```diff
@@ -170,6 +170,8 @@ class SOFA_GENERAL_ANIMATION_LOOP_API MechanicalMatrixMapper : public MixedInter
 
     Data<helper::vector<std::string>> d_forceFieldList;
 
+    unsigned int m_nbInteractionForceFields;
```

Comment:
Could you use `size_t` instead of unsigned int when you know it will store `stl container size` information.
This avoid warnings of conversion on windows64.

Reply:
Thank you. This was corrected.